cmds/core/kexec/kexec_linux.go

```diff
@@ -71,6 +73,7 @@ func registerFlags() *options {
 	flag.StringVar(&o.initramfs, "initramfs", "", "Use file as the kernel's initial ramdisk")
 	flag.BoolVarP(&o.load, "load", "l", false, "Load the new kernel into the current kernel")
 	flag.BoolVarP(&o.loadSyscall, "loadsyscall", "L", false, "Use the kexec_load syscall (not kexec_file_load)")
+	flag.StringVarP(&o.linuxImageCfgFile, "linuxImageCfgFile", "I", "", "Load Linux image from JSON info file given")
```

Comment:
Sorry, I didn't notice this before. Other flags use dashes, not camelCase -- "linux-image-cfg-file"

Reply:
Thats right, Done.